### PR TITLE
add three gird container

### PR DESCRIPTION
**Pull request by harshkotkar5557**
I have add three gird container
Preview 
three-grid

### CODE REVIEW

File: simple-grid/grid.html [hunk 1]
```diff
@@ -97,6 +98,76 @@ <h3>
...
+                                Tesla is accelerating the world's.
+                            </h3>
+                            <span>by Elon Musk</span>
+                        </div>
+                    </div>
+                    <div class="d-flex align-center justify-end">
+                        <div class="card_icons">
+                            <div class="icons_div"> <i class="fa fa-heart-o" aria-hidden="true"></i></div>
+                            <div class="icons_div">  <i class="fa fa-share-alt" aria-hidden="true"></i></div>
+                            <div class="icons_div">     <i class="fa fa-ellipsis-v" aria-hidden="true"></i></div>
```
Comment (Mahi24-prog) [inline simple-grid/grid.html:159]: Avoid extra spaces

**Review by Mahi24-prog — APPROVED** [review]
You can add more examples of grid. other than that looking good

File: simple-grid/grid.html (hunk 1 above)
Comment (harshkotkar5557) [reply simple-grid/grid.html:159]: Okay

File: simple-grid/grid.css [hunk 2]
```diff
@@ -2,16 +2,32 @@
...
-.badges_card{
+.margin-zero{
     margin: 0;
 }
+.three-grid-container{
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 1rem;
+    justify-content: center;
+    align-content: center;
```
Comment (vaibhav18matere) [inline simple-grid/grid.css:19]: add default classes you have already made.

**Review by vaibhav18matere — APPROVED** [review]

**Pull request merged**